### ISSUE TITLE
Typing problem with latest updates (4.3.0->4.3.1)

### ISSUE THREAD

**Issue by twittwer** [ngneat/spectator#192]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Behavior

Until version 4.3.0 the following code works without a problem:
```
import { createServiceFactory, SpectatorService } from '@ngneat/spectator/jest';
import { MockStore, provideMockStore } from '@ngrx/store/testing';

interface State { ... }
const initialState = { ... }

describe('MyService', () => {
    let store: MockStore<State>;

    const createService = createServiceFactory({
        service: MyService,
        providers: [provideMockStore({ initialState })],
    });
    let spectator: SpectatorService<MyService>;

    beforeEach(() => {
        spectator = createService();
        store = spectator.get<MockStore<State>>(Store);
	});
});
```

With version 4.3.1 the following line produces an error:
`store = spectator.get<MockStore<State>>(Store);`
```
Type 'SpyObject<MockStore<State>>' is missing the following properties from type 'MockStore<State>': state$, initialState, actionsObserver, reducerManager
```

I'm not sure if it's a bug or I'm doing something wrong (but it worked until 4.3.0), can someone help?

## Environment

<pre><code>
Angular version: 8.2.8

For Tooling issues:
- Node version: 10.16.1
- Platform:  Windows & Linux

Others:
- Test Runner: Jest 24.9.0
- @ngrx/store: 8.3.0
</code></pre>

**Comment by dirkluijk** [comment]
I got the same issue today. It is caused by this change: https://github.com/ngneat/spectator/commit/3d0931f. I already proposed to rollback that commit exactly because of this.

Will be fixed asap. In the meantime, please use 4.3.0.

**Comment by dirkluijk** [comment]
Should be fixed in `4.3.3`.
